utils/src/treev2/quickPickWizard/experiences/appResourceExperience.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as types from '../../../../index';
+import { QuickPickAzureSubscriptionStep } from '../quickPickAzureResource/QuickPickAzureSubscriptionStep';
+import { QuickPickGroupStep } from '../quickPickAzureResource/QuickPickGroupStep';
+import { QuickPickAppResourceStep } from '../quickPickAzureResource/QuickPickAppResourceStep';
+import { RecursiveQuickPickStep } from '../RecursiveQuickPickStep';
+import { getLastNode } from '../QuickPickWizardContext';
+import { NoResourceFoundError } from '../../../errors';
+import { AzureWizardPromptStep } from '../../../wizard/AzureWizardPromptStep';
+import { AzExtResourceType } from '../../../AzExtResourceType';
+import { AzureWizard } from '../../../wizard/AzureWizard';
+import { AzureResourceQuickPickWizardContext } from '../../../../hostapi.v2';
+import { isWrapper } from '../../../registerCommandWithTreeNodeUnwrapping';
+import { ResourceGroupsItem } from '../quickPickAzureResource/tempTypes';
+
+export async function appResourceExperience<TPick>(context: types.IActionContext, tdp: vscode.TreeDataProvider<ResourceGroupsItem>, resourceTypes?: AzExtResourceType | AzExtResourceType[], childItemFilter?: types.ContextValueFilter): Promise<TPick> {

```

Comment:
Should this support creating resources as well?

Reply:
Eventually yes. I have an API in the works to allow for it. Need to dedicate some more time to it.